### ISSUE TITLE
AVT3: Table is not recognized by JAWS

### ISSUE THREAD

**Issue by xiepingp** [ibm-cloud-docs/Monitoring-with-Sysdig#13]
=====ENVIRONMENT=====
IBM CI 162 Checkpoint Violation: IBM 1.3.1 Info and Relationships
IBM Guidance: https://www.ibm.com/able/guidelines/ci162/info_and_relationships.html
Test Tool Type: JAWS Screen Reader 2019

=====DEFECT DESCRIPTION=====
Page location: https://cloud.ibm.com/docs/Monitoring-with-Sysdig?topic=Monitoring-with-Sysdig-monitor-cf-sysdig
The table "Table 1: Metrics dictionary“ is not recognized by JAWS.

=====STEPS TO REPRODUCE=====
Keep JAWS running
-> go to https://cloud.ibm.com/docs/Monitoring-with-Sysdig?topic=Monitoring-with-Sysdig-monitor-cf-sysdig
-> Click Insert + F3 to open the Virtual HTML Features dialog
-> Click down arrow to move to Tables List, click OK button
The table "Table 1: Metrics dictionary" is not in the Table list:
![sisdig - doc - 13 table not recognized by jaws](https://user-images.githubusercontent.com/63960906/91937002-9785fb80-ed23-11ea-957c-58414ace8d6f.png)

===== EXPECTED RESULT =====
The intent of this Success Criterion is to ensure that information and relationships that are implied by visual or auditory formatting are preserved when the presentation format changes. For example, the presentation format changes when the content is read by a screen reader or when a user style sheet is substituted for the style sheet provided by the author.
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Comment by xiepingp** [comment]
@LOPEZDSR, would you please take a look at the issue? Thank you!

**Comment by vanadiscrawford** [comment]
@xiepingp I changed this single column table to a bulleted list.  I think this should resolve the JAWS issue.  Can you please confirm and close if the issue is resolved?  Thanks!

**Comment by xiepingp** [comment]
Hi @vanadiscrawford, yes, the problem no longer exists, thank you for the fixing 😄 !
![13 - fixed](https://user-images.githubusercontent.com/63960906/98195364-898b6b80-1f5c-11eb-9304-1cf73a00dfd4.png)
